server/src/internalClusterTest/java/org/opensearch/search/aggregations/bucket/SamplerIT.java

```diff
@@ -145,6 +145,7 @@ public void testIssue10719() throws Exception {
         // Tests that we can refer to nested elements under a sample in a path
         // statement
         boolean asc = randomBoolean();
+        indexRandomForConcurrentSearch("test");
```

Comment:
A few comments on this:
1. I already changed this to have multiple slices as a part of https://github.com/opensearch-project/OpenSearch/pull/11087
2. If we do want to do this we should just add it to `setupSuiteScopeCluster` once instead
3. I don't think we need to add this since the index random as implemented won't actually provide coverage of the concurrent search issues related to this test, see https://github.com/opensearch-project/OpenSearch/issues/11075